index.html

```diff
@@ -146,15 +146,15 @@ <h2>Scope</h2>
         </p>
         <ul>
           <li>Data models for verifiable credentials and presentations</li>
-          <li>Registries for the data models</li>
+          <li>Registries for extending the data models</li>
```

Comment:
iirc during the WG meeting, folks wanted to make sure we could have registries that weren't explicitly about extending the data model.

Reply:
Hrm, I'm trying to think of a registry that would fall into that category. I can't think of one, but that's probably lack of imagination on my end.

I'm happy to put it back to the way it was.

Reply:
Cool.
I don't recall the specifics, but my notes for requested changes by the group led me to remove the "extensions" language in [3369ef3](https://github.com/w3c/vc-wg-charter/commit/3369ef3ec08db7b7f42748fd581625a338fd0dd9) just last week :)